services/manuals-frontend/docker-compose.yml

```diff
@@ -0,0 +1,57 @@
+version: '3.7'
+
+x-manuals-frontend: &manuals-frontend
+  build:
+    context: .
+    dockerfile: services/manuals-frontend/Dockerfile
+  image: manuals-frontend
+  volumes:
+    - ..:/govuk:delegated
+    - bundle:/usr/local/bundle
+  working_dir: /govuk/manuals-frontend
+
+services:
+  manuals-frontend-lite:
+    <<: *manuals-frontend
+    privileged: true
+
+  manuals-frontend-app: &manuals-frontend-app
+    <<: *manuals-frontend
+    depends_on:
+      - router-app
+      - content-store-app
+      - static-app
+    environment:
+      GOVUK_ASSET_ROOT: manuals-frontend.dev.gov.uk
+      VIRTUAL_HOST: manuals-frontend.dev.gov.uk
+      HOST: 0.0.0.0
+    ports:
+      - "3000"
+    command: bin/rails s -P /tmp/rails.pid
+
+  manuals-frontend-app-draft:
+    <<: *manuals-frontend-app
```

Comment:
Is this supposed to inherit from `manuals-frontend-app`  than `manuals-frontend` like the other images?

Reply:
It's a bit of a toss up because of the inheritance, but I think I'll change it to what you suggest as it's possibly more expected.